src/net.cpp

```diff
@@ -7,6 +7,10 @@
 #include "config/bitcoin-config.h"
 #endif
 
+// must include first to ensure FD_SETSIZE is correctly set
+// otherwise the default of 64 may be used on Windows
```

Comment:
Suggest rewording to 
```
// compat.h must be included first to correctly define FD_SETSIZE
```
This is more explicit on what must be included, less unnecessarily details. Details about overriding the value 64 can go into compat if we care. Also, 64 is only default on mingw, not MSVC.